python/nx-cugraph/Makefile

```diff
@@ -15,3 +15,10 @@ lint-update:
 .PHONY: plugin-info
 plugin-info:
 	python _nx_cugraph/__init__.py
+
+objects.inv:
+	wget https://networkx.org/documentation/stable/objects.inv
```

Comment:
Rather than make this a target, should it just be done unconditionally as part of the `readme` target (and possibly removed each time)?  I'm wondering if it can go stale if left on disk unnoticed.

Reply:
Going stale is possible, but it probably changes very slowly the vast majority of the time. If it _is_ stale, CI will catch it.

Reply:
also, since I updated this to `stable` (instead of `latest`), it should only change a couple times a year.